src/components/Proposal/ResultStance.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useMemo } from 'react'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Header } from 'decentraland-ui/dist/components/Header/Header'
+
+import Cancel from '../Icon/Cancel'
+import CheckCircle from '../Icon/CheckCircle'
+
+import './ResultStance.css'
+
+interface Props {
+  resultMatches: boolean
+}
+
+function ResultStance({ resultMatches }: Props) {
```

Comment:
Does this show while the proposal is open, or only on finished proposals?

I'm thinking if we should call this `result` or `outcome` (outcome would imply the proposal is already finished)